agent/ecscni/plugin_test.go

```diff
@@ -26,7 +27,11 @@ func TestSetupNS(t *testing.T) {
 		mockResult.EXPECT().String().Return(""),
 	)
 
-	err := ecscniClient.SetupNS(&Config{})
+	additionalRoutesJson := `["169.254.172.1/32", "10.11.12.13/32"]`
+	additionalRoutes := &[]cnitypes.IPNet{}
```

Comment:
nit: this reads weird to me. Can it be just `var AdditionalLocalRoutes []cnitypes.IPNet{}` here instead and you can unmarshal it as `&additionalRoutes`?